.github/workflows/companion.yml

```
@@ -45,3 +45,32 @@ jobs:
           # Necessary for Node.js v10.x
           NODE_OPTIONS: --experimental-worker
           YARN_IGNORE_NODE: 1
+  test:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 17.x]
```

18 might make more sense?

```suggestion
        node-version: [14.x, 16.x, 18.x]
```